config/db.server.config.ts

```diff
@@ -0,0 +1,45 @@
+import database from "mongoose";
+
+/**
+ * Configure and connect to MongoDB database
+ * Uses environment variables:
+ *  - DATABASE_USER
+ *  - DATABASE_PW
+ * @param databaseName the name of the database being connected to
+ * @param isTestDatabase conditional for if the database in use is for testing
+ */
+export function connect(databaseName: string, isTestDatabase: boolean = false): Promise<typeof database> {
```

Comment:
It looks like these function parameters are never used

Reply:
yeah I overwrote some of the code so that it runs locally. Will change this before opening